qa/TL0_python-self-test-readers-decoders/test_nofw.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages="nose numpy>=1.17 opencv-python pillow librosa scipy nvidia-ml-py==11.450.51 psutil numba"
```

Comment:
Do we need all of this?

Reply:
This is good question, that I would prefer not to answer here. :V

Reply:
I hope now it runs.